example/README.md

```diff
@@ -1,9 +1,10 @@
 # Kamus Example
 A small example app, showing the power of Kamus. 
-Before running this demo, make sure Kamus is up and running, and the CLI is installed.
+Before running this demo, make sure Kamus is up and running under the namespace default, and the CLI is installed.
 
 ## Running the demo
-Before running the demo, make sure to install Kamus on the cluster. If Kamus is not installed, use the following command:
+Before running the demo, make sure to install Kamus on the cluster under the namespace default. 
+If Kamus is not installed, switch to the default namespace, if required. Then, use the following command:
```

Comment:
```suggestion
If Kamus is not installed. Then, use the following command:
```
You can pass the namespace to helm install command (`--namespace`). AFAIK there is no need to change the namespace.